akka-cluster-tools/src/multi-jvm/scala/akka/cluster/pubsub/DistributedPubSubRestartSpec.scala

```diff
@@ -103,10 +103,11 @@ class DistributedPubSubRestartSpec
       expectMsg("msg1")
       enterBarrier("got-msg1")
 
-      runOn(second) {
-        mediator ! Internal.DeltaCount
-        val oldDeltaCount = expectMsgType[Long]
+      mediator ! Internal.DeltaCount
+      val oldDeltaCount = expectMsgType[Long]
```

Comment:
previously this could have been delayed on second and first started the shutdown of third before this